sspl_test/alerts/os/test_systemd_service_actuator.py

```diff
@@ -0,0 +1,141 @@
+    service_actuator_msg = None
+    time.sleep(6)
+    ingressMsg = {}
+    while not world.sspl_modules[RabbitMQingressProcessorTests.name()]._is_my_msgQ_empty():
+        ingressMsg = world.sspl_modules[RabbitMQingressProcessorTests.name()]._read_my_msgQ()
+        time.sleep(0.1)
+        print("Received: %s " % ingressMsg)
+        try:
+            # Make sure we get back the message type that matches the request
+            msg_type = ingressMsg.get("actuator_response_type")
+            if msg_type["info"]["resource_type"] == "node:sw:os:service":
+                service_actuator_msg = msg_type
+                break
+        except Exception as exception:
+            time.sleep(0.1)
+            print(exception)
+
+    assert(service_actuator_msg is not None)
+    assert(service_actuator_msg.get("alert_type") is not None)
+    assert(service_actuator_msg.get("severity") is not None)
+    assert(service_actuator_msg.get("host_id") is not None)
+    assert(service_actuator_msg.get("info") is not None)
+    
+    info = service_actuator_msg.get("info")
+    assert(info.get("site_id") is not None)
+    assert(info.get("node_id") is not None)
+    assert(info.get("cluster_id") is not None)
+    assert(info.get("rack_id") is not None)
+    assert(info.get("resource_type") is not None)
+    assert(info.get("event_time") is not None)
+    assert(info.get("resource_id") is not None)
+
+    assert(service_actuator_msg.get("specific_info") is not None)
+
+def test_systemd_service_invalid_request(args):
+    check_sspl_ll_is_running()
+    service_actuator_request("temp_dummpy.service", "status")
+    service_actuator_msg = None
+    time.sleep(6)
+    ingressMsg = {}
+    while not world.sspl_modules[RabbitMQingressProcessorTests.name()]._is_my_msgQ_empty():
+        ingressMsg = world.sspl_modules[RabbitMQingressProcessorTests.name()]._read_my_msgQ()
+        time.sleep(0.1)
+        print("Received: %s " % ingressMsg)
+        try:
+            # Make sure we get back the message type that matches the request
+            msg_type = ingressMsg.get("actuator_response_type")
+            if msg_type["info"]["resource_type"] == "node:sw:os:service":
+                service_actuator_msg = msg_type
+                break
+        except Exception as exception:
+            time.sleep(0.1)
+            print(exception)
+
+    assert(service_actuator_msg is not None)
+    assert(service_actuator_msg.get("alert_type") is not None)
+    assert(service_actuator_msg.get("severity") is not None)
+    assert(service_actuator_msg.get("host_id") is not None)
+    assert(service_actuator_msg.get("info") is not None)
+    
+    info = service_actuator_msg.get("info")
+    assert(info.get("site_id") is not None)
+    assert(info.get("node_id") is not None)
+    assert(info.get("cluster_id") is not None)
+    assert(info.get("rack_id") is not None)
+    assert(info.get("resource_type") is not None)
```

Comment:
Better assert resource type is matching with expected one.

Reply:
Done